setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = mbdynprecice
+version = 0.0.2
```

Comment:
If we assume the previous version to be `0.1`, then this one should be `0.2`, right? Are these changes only bugfixes or significant feature changes? I guess it is anyway not stable or used enough to define a `1.0`, so we don't need to be so strict.